pfcpiface/bess.go

```diff
@@ -27,12 +32,18 @@ var (
 )
 
 type bess struct {
-	client pb.BESSControlClient
-	conn   *grpc.ClientConn
+	client             pb.BESSControlClient
+	conn               *grpc.ClientConn
+	pfcpConn           *PFCPConn
+	udpAddr            net.Addr
+	udpConn            *net.UDPConn
```

Comment:
can we remove these and use only `reportNotification` channel? Can we move the code that reads the channel into pfcp layer, keeping fastpath interface simpler?

Reply:
`readReportNotification` method will deal with backend specific ways of getting the message and put it in an agnostic way into the `reportNotification` channel looked at by pfcp/interface layer common code

Reply:
done.